src/cli/cli_udp.cpp

```diff
@@ -138,12 +138,27 @@ otError UdpExample::ProcessSend(int argc, char *argv[])
     otMessageInfo messageInfo;
     otMessage *   message = NULL;
     int           curArg  = 0;
+    uint8_t       autoGeneratedMessage[2000];
```

Comment:
We could avoid this large stack buffer by appending each octet one-by-one below. Thoughts?

Reply:
Right. Now the characters are written directly into the message buffer.